rust-1.29.yaml

```diff
@@ -0,0 +1,265 @@
+
+  - name: Build final rustc/cargo with bootstrap rustc
+    working-directory: ${{vars.mrustc-build}}
+    runs: |
+      # Inject the sysroot containing the vendored OpenSSL
+      export PKG_CONFIG_LIBDIR="${{vars.PKG_CONFIG_LIBDIR}}"
+      export CFLAGS="${{vars.CFLAGS}}"
+      export CPPFLAGS="${{vars.CPPFLAGS}}"
+      export CXXFLAGS="${{vars.CXXFLAGS}}"
+
+      export PARLEVEL=$(nproc) RUSTC_TARGET=${{host.triplet.rust}} RUSTC_VERSION=${{package.version}} MRUSTC_TARGET_VER=${{vars.short-package-version}} OUTDIR_SUF=-${{package.version}} MRUSTC=${{vars.mrustc-bin}} MINICARGO=${{vars.minicargo-bin}}
+      make -C run_rustc
+
+  - name: Unpack rustc sources (pristine)
+    working-directory: ${{vars.rustc-build}}
+    runs: |
+      tar -xzf ${{vars.mrustc-build}}/rustc-${{package.version}}-src/rustc-${{package.version}}-src.tar.gz
+      patch -p0 < /home/build/llvm-add-limits.patch
+
+  - name: Configure rustc with mrustc-built rustc
+    working-directory: ${{vars.rustc-build}}
+    runs: |
+      export PKG_CONFIG_LIBDIR="${{vars.PKG_CONFIG_LIBDIR}}"
+      export CFLAGS="${{vars.CFLAGS}}"
+      export CPPFLAGS="${{vars.CPPFLAGS}}"
+      export CXXFLAGS="${{vars.CXXFLAGS}}"
+
+      export OPENSSL_NO_VENDOR=1
+      export RUST_BACKTRACE=1
+      export ARCH=${{host.triplet.rust}}
+
+      cd rustc-${{package.version}}-src
+
+      ./configure \
+        --host="${ARCH}" \
+        --target="${ARCH}" \
+        --prefix="/usr" \
+        --local-rust-root="${{vars.mrustc-build}}/run_rustc/output-${{package.version}}/prefix" \
+        --release-channel="stable" \
+        --enable-local-rust \
+        --disable-docs \
+        --enable-extended \
+        --tools="cargo,src" \
+        --enable-option-checking \
+        --enable-locked-deps \
+        --enable-profiler \
+        --enable-vendor \
+        --python="python3" \
+        --set="rust.codegen-units=1" \
+        --set="target.${ARCH}.crt-static=false"
+
+      # Fix up config
+      sed -ri'' "s/(codegen-units =) '1'/\1 1/" config.toml
+
+  - name: Build rustc with mrustc-built rustc
+    working-directory: ${{vars.rustc-build}}
+    runs: |
+      cd rustc-${{package.version}}-src
+      sed 's/#deny-warnings = .*/deny-warnings = false/' -i config.toml
+      sed 's|deny(warnings,|deny(|' -i src/bootstrap/lib.rs
+      mkdir -p "${{targets.destdir}}/usr"
+      unset CARGO_PROFILE_RELEASE_LTO
+      unset CARGO_PROFILE_RELEASE_OPT_LEVEL
+      unset CARGO_PROFILE_RELEASE_PANIC
+      unset CARGO_PROFILE_RELEASE_CODEGEN_UNITS
+
+      export PKG_CONFIG_LIBDIR="${{vars.PKG_CONFIG_LIBDIR}}"
+      export LIBSSH2_SYS_USE_PKG_CONFIG=1
+      export CFLAGS="${{vars.CFLAGS}}"
+      export CPPFLAGS="${{vars.CPPFLAGS}}"
+      export CXXFLAGS="${{vars.CXXFLAGS}}"
+
+      export OPENSSL_NO_VENDOR=1
+      export RUST_BACKTRACE=1
+      DESTDIR=${{targets.destdir}} python3 ./x.py install --jobs $(nproc)
+
+  - uses: strip
+
+  # delete uneeded files eg uninstalltion
```

Comment:
```suggestion
  # delete uneeded files such as uninstallation scripts
```

Reply:
This is also from the original Rust build. Looks good to me.